src/Razor/test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/BasicComponent_DesignTime.codegen.cs

```diff
@@ -32,7 +32,7 @@ protected override void BuildRenderTree(Microsoft.AspNetCore.Components.RenderTr
         {
 #nullable restore
 #line 2 "TestFiles/IntegrationTests/CodeGenerationIntegrationTest/BasicComponent.cshtml"
-       __o = this.ToString();
+      __o = this.ToString();
```

Comment:
What's the root cause of these things scooting over?

Reply:
I was wondering the same thing on Friday. Hence my commit message says `(O_o)` 😄 
Investigating now.